AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1612,15 +1611,6 @@ - (void)animateLayoutTransition:(id<ASContextTransitioning>)context
       CGRect fromFrame = [context initialFrameForNode:subnode];
       CGRect toFrame = [context finalFrameForNode:subnode];
       if (CGSizeEqualToSize(fromFrame.size, toFrame.size) == NO) {
-        // To crossfade resized subnodes, show a snapshot of it on top.
-        // The node itself can then be treated as a newly-inserted one.
-        UIView *snapshotView = [subnode.view snapshotViewAfterScreenUpdates:YES];
-        snapshotView.frame = [context initialFrameForNode:subnode];
-        snapshotView.alpha = 1;
-        
-        [node.view insertSubview:snapshotView aboveSubview:subnode.view];
-        [removedViews addObject:snapshotView];
-        
         [insertedSubnodes addObject:subnode];
```

Comment:
@maicki somewhere around here, let's enable groupOpacity at least on the root node, and then set it back to its previous value at the end of the animation.

Thanks a lot for putting this up.  If you don't have time to do the groupOpacity thing now, that's totally fine, but would you be OK with landing this patch as-is?  That will help with my merge conflicts but also allow our developers on Slack to use it more easily.

The two follow ups I remember (for another diff / maybe an issue):
- Allow snapshotting as some kind of mode, suggest it be controlled in the same way that subnodes of a cell can require "no animations" from their parent.
- Always enable groupOpacity when not snapshotting